### ISSUE TITLE
bug: collapsible large title background styling

### ISSUE THREAD

**Issue by EinfachHans** [ionic-team/ionic-framework#24032]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [X] v5.x
- [X] v6.x

### Current Behavior

I'm struggling for a while with collapsible large title and background styling now.
I first though it would be only in relation with refresher: #23951 but it also is without.

Please the the attached Video from a native App (with Refresher but should be similar without):

https://user-images.githubusercontent.com/20899975/135641890-228ed15d-8b4f-4619-9eed-21e1519fe2b4.MOV

I want to be able to have a complete background colored header (including the large title). If i scroll down (where the refresher comes up or if there is no one, the title gets a small step bigger) the background should be in the same color.

Currently when creating a nearly similar look by setting the color on the main toolbar directly, there comes up this gap:
<img src="https://user-images.githubusercontent.com/20899975/136371628-8fa37e7c-0e06-427b-a0ed-edb3dc104d16.png" width="300">

### Expected Behavior

I think we have to decide if we want show the content above the header in expanded state. I saw many native apps that look like the video above and i really like this look 😃 

In that case the opacity should not be set to `0` and the gap has to be filled somehow 🤔

### Steps to Reproduce

I guess no reproduction is needed here and this is more kind of a discussion ticket if such a functionality should find it way into the framework (i hope so) 😃

**Comment by liamdebeasi** [comment]
Can you provide a code example of how you are currently styling the large title?

**Comment by EinfachHans** [comment]
Yeah i understand but i find it's kinda sad that i'm not able to change that styling behaviour. On native i'm able to style it that way, so it would be great to be possible here too.

Wrapping in a div is also not working 100% as you can see the background on bottom when you scroll too fast or too much. Can you think of a 100% working workaround or a way of implementing this into the framework as a kind of feature/separate styling?
